.pipelines/cni/load-test-templates/create-cluster-template.yaml

```diff
@@ -4,6 +4,7 @@ parameters:
   nodeCount: ""
   vmSize: ""
   vmSizeWin: ""
+  osSKU: ""

```

Comment:
nit: Can we default this to Ubuntu? If there is a case where it needs to be specified it can be passed in.